chef/automate
# Fetch mount location

### :nut_and_bolt: Description: What code changed, and why?
In this Pr we trying to get mount location using df -h

### :chains: Related Resources
https://chefio.atlassian.net/browse/CHEF-2234
### :+1: Definition of Done

### :athletic_shoe: How to Build and Test the Change
1. run `hab studio enter` locally
2. rebuild `components/automate-cli`
3. get bin from pkg to src `cp /hab/pkgs/smudassi/automate-cli/0.1.0/<time stamp>/bin/chef-automate .`
4. upload binary file into ec2 instance
5. make sure to setup nfs on ec2 instance
6. run `chef-automate verify serve`
7. open new terminal ssh into your instance
8. run the curl request
`curl --location --request POST 'http://localhost:7799/api/v1/fetch/nfs-mount-loc' --header 'Content-Type: application/json' --data-raw '{
  "mount_location": "/mnt/nfs"
}' | jq`


### :white_check_mark: Checklist

**All PRs** must tick these:


With occasional exceptions, all PRs **from Progress employees** must tick these:


All PRs **from Progress employees** should tick these if appropriate:


*Please add a note next to any checkbox above if you are NOT ticking it.*
### :camera: Screenshots, if applicable
**/api/v1/fetch/nfs-mount-loc** test
<img width="1507" alt="Screenshot 2023-06-02 at 3 55 39 PM" src="https://github.com/chef/automate/assets/47217471/3d649d07-a966-481b-9597-802f11c0b719">

**/api/v1/checks/nfs-mount** test
<img width="1490" alt="Screenshot 2023-06-02 at 4 27 20 PM" src="https://github.com/chef/automate/assets/47217471/0215d16f-9f89-420e-960e-4b6021f2377c">


<img width="1022" alt="Screenshot 2023-05-23 at 12 15 24 PM" src="https://github.com/chef/automate/assets/47217471/c1a0ef9f-0544-4d4a-bbef-41069903088b">

<img width="945" alt="Screenshot 2023-05-23 at 11 28 24 AM" src="https://github.com/chef/automate/assets/47217471/3dccc829-e2ee-4955-8a48-62385cb9da0d">
